writeengine/shared/we_brm.cpp

```diff
@@ -1134,234 +1135,243 @@ int BRMWrapper::rollBackBlocks(const VER_t transID, int sessionId)
 
   std::vector<BRM::FileInfo> files;
 
-  for (i = 0; i < lbidList.size(); i++)
+  try
```

Comment:
I missed the part where the patch looks for rollBackBlocks() result code. Could you point me to this line/-s?

Reply:
It throws exception and we handle it https://github.com/mariadb-corporation/mariadb-columnstore-engine/pull/2655/files#diff-3da43348b5fbf465a6af2ecfd724226a5450af15fab2204485a9f1ad15358ad1R1371

Reply:
The return code is handling here https://github.com/mariadb-corporation/mariadb-columnstore-engine/pull/2655/files#diff-ff20b84c0c434c434ccb22533ef5e7810bb9cc4603e6b002447c931479bf5c3dR274 the problem there it's a 8 bit value, I didnt wanted to change it.